js/content-script.js

```diff
@@ -162,7 +162,7 @@ function overrideRequireJs(b, extensionId) {
           if (p) {
             var jb = r(),
               kb = 0;
-            if (ib != null) kb = hb.stopTiming(ib);
+            if (ib !== null) kb = hb.stopTiming(ib);
```

Comment:
@J2TeaM I want to keep all the code from line 21 to line 457 beacause they're from Fb's code.